### ISSUE TITLE
Missing zh_TW.json in Intl/Resources/data/regions

### ISSUE THREAD

**Issue by lanphy** [symfony/symfony#21457]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | yes
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.2.2

Hi, in the Symfony\Component\Intl\Resources\data\regions directory, there is a missing zh_TW.json file for the list of countries in traditional Chinese caracters (for Taïwan).
I add mine in attach, if that may helps. [zh_TW.zip](https://github.com/symfony/symfony/files/737772/zh_TW.zip)
Thanks,
Lanphy

**Comment by javiereguiluz** [comment]
In the ICU data, `zh_TW` is defined as an alias:

```
zh_TW{
    "%%ALIAS"{"zh_Hant_TW"}
}
```

Maybe there's some issue with aliases? Let's ask to our ICU/Intl expert @jakzal. Thanks!

**Comment by xabbuh** [comment]
@jakzal Any chance you can help here? :)

**Comment by jakzal** [comment]
@lanphy what's the problem you're experiencing? Would you be able to write a snippet of code reproducing the issue? 

Is the result of the script below not what you expect?

```php
<?php

require_once __DIR__.'/vendor/autoload.php';

use Symfony\Component\Intl\Intl;

\Locale::setDefault('zh_TW');

$countries = Intl::getRegionBundle()->getCountryNames();

var_dump($countries);
```

**Comment by lanphy** [comment]
Hi, I'm not experiencing an "issue", I'm just notifing that there is a missing file in Symfony package (zh_TW.json), that I have added as attached file here, and offer my suggest to add it in the package.

**Comment by jakzal** [comment]
We can't just upload the file you sent us, as all of these files are generated out of data provided by ICU. 
I'm also not sure where you got the file, since in the most recent ICU version [zh_TW is an alias](http://source.icu-project.org/repos/icu/tags/release-58-2/icu4c/source/data/region/zh_TW.txt) to another locale.

This is why I'm asking if there's any issue you've experienced. Is the list of countries wrong for `zh_TW`? Does it show data from wrong locale? Otherwise there's nothing to fix here.

**Comment by stof** [comment]
@lanphy the json files are not meant to be accessed directly. They are used as source by the API of the component. So any fix should be related to an issue with the component API, not just related to looking at the list of files

**Comment by lanphy** [comment]
I see. What I experienced is : there is not traditionnal Chinese (Chinese Taiwan) language for the list of countries, only a simplified version one.

**Comment by jakzal** [comment]
@lanphy which list are you talking about exactly?

**Comment by lanphy** [comment]
The CountryType Field.

**Comment by jakzal** [comment]
I just verified and Taiwan is on the list of countries.

**Comment by lanphy** [comment]
No, I mean there isn't "Traditionnal Chinese" in the list of translated languages.

**Comment by jakzal** [comment]
I can see it on the list coming from ```Intl::getLanguageBundle()->getLanguageNames()```:

```
    [zh_Hant] => Traditional Chinese
```

**Comment by lanphy** [comment]
Yes, but the content seems missing.

**Comment by jakzal** [comment]
@lanphy you really need to be more specific. What are you trying to do? What's the actual and the expected behaviour?

**Comment by lanphy** [comment]
When I want to display the list of countries with CountryType in zh_Hant_TW language, it only displays zh_Hant. So I looked into the library, and as I guessed, there is missing all the zh_Hant_TW labels for the list of countries.

**Comment by jakzal** [comment]
Ok, could you show me an example of the actual value vs the expected value in the country select? A single key and a single value will do. For example: I see "a => b", but I'd like to see "c => d".

**Comment by lanphy** [comment]
Instead of "AD": "安道爾", I get "AD": "安道尔".

**Comment by lanphy** [comment]
If you don't see any issue, it might be my implementation which is not correct.

**Comment by jakzal** [comment]
@lanphy thanks, with your help I managed to reproduce the issue. For `zh_Hant_TW` I see `安道爾`, but for `zh_TW` I see `安道尔` (while the first version should be shown in both cases). So the alias doesn't seem to work properly. I yet need to determine if it's on our side, or ICU.

**Comment by fabpot** [comment]
@jakzal Do you have time to check if the issue is on our side? Thanks.

**Comment by jakzal** [comment]
Just did some more debugging and looks like it's on our side. I'm looking into the best way to fix it.

**Comment by jakzal** [comment]
Fixed by #26452.
